sdks/python/apache_beam/options/pipeline_options_validator.py

```diff
@@ -92,6 +94,8 @@ class PipelineOptionsValidator(object):
   ERR_INVALID_PROJECT_ID = (
       'Invalid Project ID (%s). Please make sure you specified the Project ID, '
       'not project description.')
+  ERR_INVALID_ENDPOINT = (
+      'Invalid url (%s) for dataflow endpoint. Please provide a valid url.')
```

Comment:
Can we make it more actionable like enpoint url examples to let users know what format Beam expects?

Reply:
Beam doesn't expect anything more specific. We only test for a valid url that starts with 'http' or 'https'.

Reply:
I see.